src/01-functions.js

```diff
@@ -39,7 +43,9 @@ function greet() {}
  *  add(10, 20);
  *  //> 30
  */
-function add() {}
+function add(a, b) {
+return (a + b);  
```

Comment:
Lets indent our return statement over one tab here.  Also we do not need the ` ( ) ` in this case.